protocol-configuration/proposalsConfig.ts

```diff
@@ -3,49 +3,59 @@ import tip_111 from '@proposals/description/tip_111';
 import pod_exec_v2 from '@proposals/description/pod_exec_v2';
 import tokemak_withdraw from '@proposals/description/tokemak_withdraw';
 import tip_112 from '@proposals/description/tip_112';
+import timelock_admin from '@proposals/description/timelock_admin';
 
 const proposals: TemplatedProposalsConfigMap = {
-  tip_111: {
+  // tip_111: {
```

Comment:
Think this is leftover commenting and needs deleting? `tokemak_withdraw` and `tip_112` can both be removed.

`pod_exec_v2` needs to stay here, it hasn't been proposed/executed by the TC yet